README.md

```diff
@@ -111,3 +111,91 @@ Quering schema field types (the order matches `get_names`):
 ```lua
 avro_schema.get_types(schema)
 ```
+
+## References
+
+Named types are ones that have mandatory `name` field in its definition:
+record, fixed, enum.
+
+Named types can be referenced after the first definition (forward declarations
+are forbidden). Of course, primitive types can be referenced in any place of a
```

Comment:
Thay will be allowed soon.
Do you want to change the text afterward or make it an undocumented feature?

Reply:
I think we need to describe the current state and change the description with new ehancements.